umbraco/Umbraco.CMS.Backoffice
# Bugfix: Persist User document and media start nodes

## Description

This PR updates the persisting user start nodes to support the new backend models. The PR also aligns the UI to include a toggle to access all documents/media like in the user group workspace.

> [!IMPORTANT]  
> Merges together with: https://github.com/umbraco/Umbraco-CMS/pull/16094

Missing:
* The start node access is not enforced on the user yet. This PR only updates persisting the data.

## Types of changes




## How to test?

Run the backend from this PR: https://github.com/umbraco/Umbraco-CMS/pull/16094

Make sure you can (User workspace):
* select a document start nodes(s) and persist it
* give access to all documents
* select a media start node(s) and persist it
* give access to all media
